test/CMakeLists.txt

```diff
@@ -113,3 +125,29 @@ add_custom_target(test-force
     COMMAND ${PROJECT_SOURCE_DIR}/test/test-run.py
         --builddir=${PROJECT_BINARY_DIR}
         --force)
+
+message(STATUS "Add integration tests")
+file(GLOB_RECURSE tests ${PROJECT_SOURCE_DIR}/tests *.test.lua *.test.py *_test.lua)
```

Comment:
There is no `tests` directory in the repository (it is `test`). Is it a typo?

Reply:
Right, it was a typo. Fixed:

```diff
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -178,8 +178,8 @@ list(APPEND EXECUTED_WITHOUT_TEST_RUN
     vinyl-luatest
 )
 
-message(STATUS "Add integration tests")
-file(GLOB_RECURSE tests ${PROJECT_SOURCE_DIR}/tests *.test.lua *.test.py *_test.lua)
+file(GLOB_RECURSE tests ${PROJECT_SOURCE_DIR}/test
+  *.test.lua *.test.py *_test.lua)
 foreach(test_path ${tests})
   get_filename_component(test_name ${test_path} NAME)
   get_filename_component(basedir ${test_path} DIRECTORY)
```